components/common/mojomUtils.ts

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) 2021 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// you can obtain one at http://mozilla.org/MPL/2.0/.
+
+import * as mojo from 'gen/mojo/public/mojom/base/time.mojom.m.js'
+
+/**
+ * Converts a mojo time to a JS time.
+ */
+export function mojoTimeToJSDate (mojoTime: mojo.Time): Date {
```

Comment:
How/where does this get called?

Reply:
Ah - I see; this *can* be used (just didn't see an example in the PR description)
> I also included a utility for importing and converting mojom's `Time` type to JS's `Date` type.

Using the example there in the first screenshot (from PR description), you could get the value for `a.publish_time` and then pass it in here, to convert 👍

Reply:
@bsclifton that's right, sorry for the confusion - I call it in my separate PR but since that might not be merged for a while I thought others who use mojom and will benefit from this PR might need it. And also wanted to include _some_ usage of something from gen/